src/inspector_agent.cc

```diff
@@ -219,8 +214,32 @@ bool RespondToGet(InspectorSocket* socket, const std::string& script_name_,
   return true;
 }
 
-}  // namespace
+std::string StringViewToUtf8(const StringView& view) {
+  if (view.is8Bit())
+    return std::string(reinterpret_cast<const char*>(view.characters8()),
+        view.length());
+
+  // Reserve 1 character for terminator.
+  std::string result(view.length() * sizeof(UChar) + 1, '\0');
```

Comment:
`sizeof(UChar)` doesn't look self-evidently correct.  I assume it's shorthand for `sizeof(*view.characters16())` (which is `sizeof(uint16_t)`) but code points >= 0x800 encode to more than two bytes.